### DISCUSSION TITLE
Time range using time range?

### DISCUSSION THREAD

**Discussion by kronicklez** [akvorado/akvorado#1526] [Q&A] [answered]
Hi @vincentbernat / expert,

Is it possible we do search time range for example we want to see traffic from 10am to 11am. 

Thanks and appreciate your feedback

**Comment by vincentbernat** [comment, marked as answer]
- Start: today at 10am
- End: today at 11am

**Reply by kronicklez** [reply]
Hi @vincentbernat ,

Noted. Many thanks. Appreciate your fast feedback
